scripts/base/frameworks/broker/main.zeek

```diff
@@ -385,9 +404,41 @@ event Broker::log_flush() &priority=10
 	schedule Broker::log_batch_interval { Broker::log_flush() };
 	}
 
+function update_metrics_exporter_interval(id: string, val: interval): interval
+	{
+	Broker::__set_metrics_exporter_interval(val);
+	return val;
+	}
+
+function update_metrics_exporter_target(id: string, val: string): string
+	{
+	Broker::__set_metrics_exporter_target(val);
+	return val;
+	}
+
+function update_metrics_exporter_id(id: string, val: string): string
+	{
+	Broker::__set_metrics_exporter_id(val);
+	return val;
+	}
+
+function update_metrics_exporter_prefixes(id: string, filter: vector of string): vector of string
+	{
+	Broker::__set_metrics_exporter_prefixes(filter);
+	return filter;
+	}
+
 event zeek_init()
 	{
 	schedule Broker::log_batch_interval { Broker::log_flush() };
+	Option::set_change_handler("Broker::metrics_exporter_interval",
```

Comment:
Based on description of change handlers from https://docs.zeek.org/en/master/frameworks/configuration.html#when-change-handlers-trigger, think an explicit call to each individual `update_metrics_` function is warranted at this point to ensure those actually consistently get set at startup-time to what a user expects.

Reply:
Done.